RELEASE_NOTES.md

```diff
@@ -24,6 +24,77 @@ Chef now ships with Ruby 2.6.2. This new version of Ruby improves performance an
 
 ## Breaking Changes
 
+### Knife Bootstrap
+
+Knife bootstrap has been updated, and Windows bootstrap has been merged in Knife core bootstrap. This marks the deprecation of knife-windows plugin's `bootstrap` behavior.
```

Comment:
Does "Knife core bootstrap" require some sort of denotation (`Knife core bootstrap`) or hyphen (Knife-core bootstrap)? Asking since I'm not sure what previous formatting has been used for this.

Reply:
It looks like the words I should use here are "into core Knife" based on past release notes which talk about merging cookbook resources into "core Chef".

Reply:
Nope, that's not right - Chef refers to the product name; knife is just a component...

Reply:
That leaves me with "into core Chef's `knife bootstrap`"